### PR TITLE
Add Peek

### PR DESCRIPTION
**Pull request by chriszielinski**
Hi Sindre. 👋

Would love to see Peek added to the list!

### CODE REVIEW

**Comment by lovetoburnswhen** [comment]
t'isnt free

**Comment by chriszielinski** [comment]
> t'isnt free

Unfortunately, this is true. I did not see anything mentioning free as a prerequisite. Let me know if I am mistaken. I added the 💰 in the title to distinguish the fact that it is paid & use "purchase" as the call to action to indicate so. I felt this was appropriately transparent. I did leave out the price as this depends on the locality.

File: readme.md [hunk 1]
```diff
@@ -126,6 +126,14 @@ Run `brew install qlvideo` or [download manually](https://github.com/Marginal/QL
 
 *These are not included in [Install all](#install-all).*
 
+### [Peek](https://bigzlabs.com/peek) 💰
+
+> Peek delivers Quick Look functionality you will not find anywhere else on this planet to over 300 file extensions: copy and find text, jump to line/page numbers, Github-flavored Markdown with a generated table of contents, syntax highlighting, scroll restoration, and more.
```
Comment (sindresorhus) [inline readme.md:131]: Can you make it less like a marketing tagline? I mean this part "you will not find anywhere else on this planet"

**Comment by sindresorhus** [comment]
The image needs to follow the same style as the others.

**Comment by sindresorhus** [comment]
The fact that it's paid is not a problem.

**Comment by chriszielinski** [comment]
No problem. Let me know if the changes are satisfactory.

**Pull request merged**